src/main/java/edu/kis/vh/nursery/stack/Collections.java

```diff
@@ -0,0 +1,19 @@
+package edu.kis.vh.nursery.stack;
+
+public interface Collections {

```

Comment:
Names of operations do not correspond to operating the collection, thus do not reflect the interface name.
Name of the interface should not be plular